packages/framework/src/shared/common.ts

```diff
@@ -33,7 +33,7 @@ export const findExpressions = (
   Object.entries(obj).forEach(([key, value]) => {
     const curPath = path.concat(key);
     if (isExpression(value)) {
-      expressionMap.push([curPath.join("."), combineExpressions(value)]);
+      expressionMap.push([curPath.join("."), `\${\`${value || ""}\`}`]);
```

Comment:
You only interpolate as string if it's a multiple expression, otherwise if this returns a number or object you will create issues.